test/lexers/jsx-lexer.test.js

```diff
@@ -233,10 +233,10 @@ describe('JsxLexer', () => {
     it('erases tags from content', (done) => {
       const Lexer = new JsxLexer()
       const content =
-        '<Trans>a<b test={"</b>"}>c<c>z</c></b>{d}<br stuff={y}/></Trans>'
```

Comment:
Why did you modify theses two tests? Leaving `{d}` in there is more informative don't you think?

Reply:
Hmm, I suppose so. Would you like me to assert that a warning is emitted in these tests as well? Or should I just undo the changes?

Reply:
I usually prefer tests that check for a single thing as they are easier to maintain but I don't have a strong opinion either way.

Reply:
Okay, I tend to agree. I'll leave it out in that case.